contracts/truefi2/SAFU.sol

```diff
@@ -25,6 +25,8 @@ contract SAFU is UpgradeableClaimable {
         require(loanFactory.isLoanToken(address(loan)), "SAFU: Unknown loan");
         require(loan.status() == ILoanToken2.Status.Defaulted, "SAFU: Loan is not defaulted");
         ITrueFiPool2 pool = ITrueFiPool2(loan.pool());
-        IERC20(pool.token()).safeTransfer(address(pool), loan.debt());
+        pool.liquidate(loan);
+        uint256 lostByPool = loan.debt().mul(loan.balanceOf(address(this))).div(loan.totalSupply());
+        IERC20(pool.token()).safeTransfer(address(pool), lostByPool);
```

Comment:
I think pool should be a little more distrustful towards SAFU there. Maybe we could approve pool for `lostByPool` amount of tokens and then `pool.liquidate(loan)` would handle `transferFrom`?